src/ui/src/composables/useWriterTracking.ts

```diff
@@ -38,7 +38,8 @@ type WriterTrackingEventName =
 	| "blueprints_run_succeeded"
 	| "blueprints_logs_trace_opened"
 	| "blueprints_new_added"
-	| "blueprints_block_output_copied";
+	| "blueprints_block_output_copied"
+	| "shared_blueprint_added";
```

Comment:
I think you should prefix by `blueprints_`

```suggestion
	| "blueprints_shared_blueprint_added";
```